src/ott/neural/networks/velocity_field.py

```diff
@@ -52,20 +52,23 @@ class VelocityField(nn.Module):
   time_encoder: Callable[[jnp.ndarray],
                          jnp.ndarray] = time_encoder.cyclical_time_encoder
   act_fn: Callable[[jnp.ndarray], jnp.ndarray] = nn.silu
+  dropout_rate: float = 0.0
 
   @nn.compact
   def __call__(
       self,
       t: jnp.ndarray,
       x: jnp.ndarray,
       condition: Optional[jnp.ndarray] = None,
+      deterministic: bool = False
```

Comment:
Think it's better to use the `train: bool = True` naming (+ flipping the logic), more explicit.